<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyślij</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarRpm">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="Polaczenie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Połączenie</string>
    </property>
    <widget class="QPushButton" name="pushButtonSearch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>23</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Szukaj</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLink">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>22</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Połącz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClose">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>22</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Rozłącz</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDevices">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>23</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEditLogi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
